spec/swagger_v2/mount_override_api_spec.rb

```diff
@@ -25,23 +25,12 @@ def app
     end
 
     Class.new(Grape::API) do
-      mount old_api
       mount new_api
+      mount old_api
```

Comment:
What prompted this change?  While I can understand if we want to keep these tests decoupled from tastytrade-specific practices, our policy is to mount the latest version last, like here: https://github.com/tastyworks/account-positions-api/blob/master/app.rb

https://project-soho.atlassian.net/wiki/spaces/SOHO/pages/2453471645/Versioning+Strategy

Reply:
Just saw your other PR https://github.com/tastyworks/grapeyworks/pull/225, I see chronological order isn't being validated anymore though I'm curious as to why.  I'll keep an eye on https://github.com/tastyworks/grapeyworks/pull/225#discussion_r1899733360